sources/packages/web/src/views/institution/locations/active-applications/ActiveApplicationSummaryData.vue

```diff
@@ -0,0 +1,228 @@
+<template>
+  <body-header
+    title="Applications"
+    :recordsCount="applications.results?.length"
+    class="m-1 mb-12"
+  >
+    <template #actions>
+      <div class="search-box-display-width">
+        <v-text-field
+          label="Search name or application #"
+          prepend-inner-icon="mdi-magnify"
+          variant="outlined"
+          v-model="searchCriteria"
+          @keyup.enter="searchActiveApplications"
+        ></v-text-field>
+      </div>
+    </template>
+  </body-header>
+  <content-group>
+    <toggle-content :toggled="!applications.results?.length">
+      <DataTable
+        :value="applications.results"
+        :lazy="true"
+        class="p-m-4"
+        :paginator="true"
+        :rows="pageLimit"
+        :rowsPerPageOptions="rowsPerPageOptions"
+        :totalRecords="applications.count"
+        @page="pageEvent"
+        @sort="sortEvent"
+      >
+        <template #empty>
+          <p class="text-center font-weight-bold">No records found.</p>
+        </template>
+        <Column
+          field="fullName"
+          header="Name"
+          :sortable="true"
+          headerStyle="width: 20%"
+        >
+          <span>{{ data.fullName }}</span>
```

Comment:
please use `template` like this 
![image](https://user-images.githubusercontent.com/77353155/174657905-28427674-3393-41bf-b4a5-aae9ab828da0.png). same for line 28